### ISSUE TITLE
Is there a way/a command to quickly reopen install/upgrade messages of a package?

### ISSUE THREAD

**Issue by dubeg** [wbond/package_control#960]
I quickly checked; there doesn' seem to be any way to reopen a message of a package. Sometimes, it can contain useful infos about a package that are easy to forget when time goes by.

**Comment by wbond** [comment]
No, there isn't other than visiting the package repository and looking for messages.

Can you think of how you would like such a feature to work? Would you want to see a list of all messages and be able to open them one at a time? Would you want it to automatically open all upgrade messages and the install message?

**Comment by dubeg** [comment]
Hm. I've been wondering how to achieve this. Perhaps when you execute the 'List Packages' command , then select a package entry, you could end up with a list its messages. From there, selecting one message would open it in the editor. What do you think?
